cmd/run.go

```diff
@@ -440,21 +443,39 @@ a commandline interface for interacting with it.`,
 			logrus.Warn("No data generated, because no script iterations finished, consider making the test duration longer")
 		}
 
+		data := ui.SummaryData{
+			Metrics:   engine.Metrics,
+			RootGroup: engine.Executor.GetRunner().GetDefaultGroup(),
+			Time:      engine.Executor.GetTime(),
+			TimeUnit:  conf.Options.SummaryTimeUnit.String,
+		}
 		// Print the end-of-test summary.
 		if !conf.NoSummary.Bool {
 			fprintf(stdout, "\n")
 
 			s := ui.NewSummary(conf.SummaryTrendStats)
-			s.SummarizeMetrics(stdout, "", ui.SummaryData{
-				Metrics:   engine.Metrics,
-				RootGroup: engine.Executor.GetRunner().GetDefaultGroup(),
-				Time:      engine.Executor.GetTime(),
-				TimeUnit:  conf.Options.SummaryTimeUnit.String,
-			})
+			s.SummarizeMetrics(stdout, "", data)
 
 			fprintf(stdout, "\n")
 		}
 
+		if conf.SummaryExport.ValueOrZero() != "" {
+			f, err := os.Create(conf.SummaryExport.String)
+			if err != nil {
+				logrus.WithError(err).Error("failed to create summary export file")
+			} else {
+				defer func() {
+					if err := f.Close(); err != nil {
+						logrus.WithError(err).Fatal("failed to close summary export file")
```

Comment:
`logrus.Fatal()` aborts the k6 execution - I'm not sure that's needed for simply failing to close a file at the end of the test run. This should be `logrus.WithError(err).Error()`

Reply:
Fixed